applications/app/views/fragments/galleryBody.scala.html

```diff
@@ -66,8 +66,8 @@ <h2 class="fc-container__header__title">
                     itemprop="contentUrl" />
 
                 <span class="rounded-icon gallery2__fullscreen">
-                    <i class="i i-expand-white"></i>
-                    <i class="i i-expand-black"></i>
+                    @fragments.inlineSvg("expand-white", "icon", List("i"))
```

Comment:
given we already know it's an icon, it seems overkill to use the old `.i` class. several former `.i` elements ahve been moved across so far, fancy digging around and seeing how they handled it?

if it turns out we just havn't moved any that took advantage of the centering, it might be better to create a `.centered-icon` class, for example, alongside `.rounded-icon`?